### DISCUSSION TITLE
bash: ../../../devel/setup.bash: No such file or directory

### DISCUSSION THREAD

**Discussion by ElaineYao** [USC-ACTLab/crazyswarm#627] [Q&A] [answered]
Hi,

I'm following these steps to install Crazyswarm. 
```
$ cd crazyswarm
$ ./buildSimOnly.sh
$ cd ros_ws/src/crazyswarm/scripts
$ source ../../../devel/setup.bash
```
However, after I execute the last command, it gave me this error, " **bash: ../../../devel/setup.bash: No such file or directory**
"
I checked and there is indeed no folder called `devel` under `src`. I was wondering what could be wrong with my environment? Thanks,

**Comment by jpreiss** [comment, marked as answer]
you need to run the full `build.sh` if you want to use ROS.
